src/components/LanguageWrapper/index.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import { IntlProvider } from 'react-intl';
+import { useSelector, shallowEqual, useDispatch } from 'react-redux';
+
+import { preferencesSetLocale } from 'state/actions/preferences';
+import english from 'languages/en';
+import spanish from 'languages/es';
+import { availableLocales } from 'utils/index';
+
+const browserLocale = navigator.language.split(/[-_]/)[0];
+
+const messages = {
+  en: english,
+  es: spanish
+};
+
+export const languages = availableLocales(messages);
```

Comment:
Why are we exporting this? Instead we should call `availableLocales` where we need it.